src/ucs/sys/string.h

```diff
@@ -75,6 +75,24 @@ uint64_t ucs_string_to_id(const char *str);
 void ucs_memunits_to_str(size_t value, char *buf, size_t max);
 
 
+/**
+ * Convert a string holding memory units to a numeric value.
+ *
+ *  @param buf   String to convert
+ *  @param dest  Numeric value of the string
+ *  @param arg   Argument to use
+ */
+int ucs_str_to_memunits(const char *buf, void *dest, const void *arg);
```

Comment:
need to make this return ucs_status_t - it's not in config parse any more. and why do we need *arg?